### DISCUSSION TITLE
Topic and Routing key

### DISCUSSION THREAD

**Discussion by Sevolith** [taskiq-python/taskiq#246] [Q&A]
Using rabbitMQ I would like to know how i can use the routing key functionality with topic exchange ? 

Thanks you

**Comment by s3rius** [comment]
Hi and thanks for asking. But what do you want to achieve?

**Reply by s3rius** [reply]
The most obvious thing that you can achieve is: you can listen to specific tasks from other services. 

For example, if service A uses taskiq, some service B might listen to specific tasks to keep track of something.
